eastwood.clj

```diff
@@ -8,3 +8,7 @@
 (disable-warning
  {:linter :deprecations
   :symbol-matches #{#"public final void java.lang.Thread.stop\(\)"}})
+
+(disable-warning
```

Comment:
I'm not very familiar with Eastwood, so I'm not sure whether this is a sensible way to disable warnings for the new tests.